tools/runtime-stats-monitor.md

```diff
@@ -0,0 +1,215 @@
+---
+id: RNSM
+title: Runtime Network Stats Monitor
+sidebar_label: Runtime Network Stats Monitor
+---
+
+The **Runtime Network Stats Monitor (RNSM)** is a configurable component to display various network stats in an on-screen display at runtime, including stats from the Netcode for GameObjects (Netcode) package and custom, user-defined stats.
+
+## Prerequisites
+
+To enable the Runtime Network Stats Monitor, [install the Multiplayer Tools package](install-tools.md). 
+
+You should now see the Runtime Network Stats Monitor from the **Unity Hub** top menu bar under **Component** > **Netcode**, or when searching for it by name after pressing **Component** > **Add..**. 
+
+:::note
+You must have an object selected for these options to be available.
+:::
+
+## Adding the Runtime Network Stats Monitor to a Scene
+
+To add the RNSM to your scene, you must add it as a component to an object:
+
+1. Select an object in your scene. <- SHOULD AN OBJECT BE MADE SPECIFICALLY FOR THE RNSM? LIKE BEST PRACTICE-WISE? EMPTY OBJECT OR A SPECIFIC TYPE
+2. Add the RNSM commponent by either:
+   1. Selecting **Add Component** in the **Inspector** tab, navigate to **Netcode** and select **RuntimeNetStatsMonitor**.
+   OR
```

Comment:
When selecting `Add Component` there should be a search bar that pops up, which is by far the easiest way of adding the RNSM 🙂